src/BenchmarksDriver/Program.cs

```diff
@@ -25,60 +25,52 @@ public class Program
-                { Scenario.MvcJson, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, Headers = _jsonHeaders
-                } },
-                { Scenario.MemoryCachePlaintext, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
-                } },
-                { Scenario.MemoryCachePlaintextSetRemove, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
-                } },
-                { Scenario.ResponseCachingPlaintextCached, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
-                } },
-                { Scenario.ResponseCachingPlaintextResponseNoCache, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
-                } },
-                { Scenario.ResponseCachingPlaintextRequestNoCache, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders.Append("Cache-Control: no-cache")
-                } },
-                { Scenario.ResponseCachingPlaintextVaryByCached, new ClientJob() {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
+                { Scenario.Plaintext, new ClientJob(_plaintextJob) },
+                { Scenario.Json, new ClientJob(_jsonJob) },
+                { Scenario.CopyToAsync, new ClientJob(_plaintextJob) {
+                    ScriptName = "post",
                 } },
-                { Scenario.StaticFiles, new ClientJob {
-                    Connections = 256, Threads = 32, Duration = 15, ScriptName = "pipeline", PipelineDepth = 16, Headers = _plaintextHeaders
+                { Scenario.MvcPlaintext, new ClientJob(_plaintextJob) },
+                { Scenario.MvcJson, new ClientJob(_jsonJob) },
+                { Scenario.MemoryCachePlaintext, new ClientJob(_plaintextJob) },
+                { Scenario.MemoryCachePlaintextSetRemove, new ClientJob(_plaintextJob) },
+                { Scenario.ResponseCachingPlaintextCached, new ClientJob(_plaintextJob) },
+                { Scenario.ResponseCachingPlaintextResponseNoCache, new ClientJob(_plaintextJob) },
+                { Scenario.ResponseCachingPlaintextRequestNoCache, new ClientJob(_plaintextJob) {
+                    Headers = _plaintextJob.Headers.Append("Cache-Control: no-cache")
                 } },
+                { Scenario.ResponseCachingPlaintextVaryByCached, new ClientJob(_plaintextJob) },
+                { Scenario.StaticFiles, new ClientJob(_plaintextJob) },
```

Comment:
For the many entries where the `CleintJob` is exactly `_plaintextJob` or `_jsonJob`, why make copies? Changes made later are either done after creating a copy or applied to all entries in this dictionary. This is minor from a performance perspective but makes the code harder to read.

Reply:
Good suggestion.  Fixed.